.gitignore

```diff
@@ -31,9 +31,6 @@
 /coverage/*
 /client/coverage/*
 
-# Ignore additional themes
```

Comment:
Leave this in. We don't change the default theme much and if you're working on additional themes you don't want to accidentally commit them to the main Coursemology repo.

Reply:
It is possible to include just the default folder though:

https://stackoverflow.com/questions/5533050/gitignore-exclude-folder-but-include-specific-subfolder

Reply:
Ok let's go with Jeremy's suggestion.